crds/workspace.devfile.io_devworkspaces.v1beta1.yaml

```diff
@@ -231,10 +231,11 @@ spec:
                           commandLine:
                             description: "The actual command-line string \n Special
                               variables that can be used: \n  - `$PROJECTS_ROOT`:
-                              A path where projects sources are mounted \n  - `$PROJECT_SOURCE`:
+                              A path where projects sources are mounted as defined
+                              by container component's sourceMapping \n  - `$PROJECT_SOURCE`:
                               A path to a project source ($PROJECTS_ROOT/<project-name>).
                               If there are multiple projects, this will point to the
-                              directory of the first one."
+                              directory of the first one"
```

Comment:
Missing ending period

Reply:
I've checked online and the general rule is; complete sentences should have periods and fragmented sentences should not have periods. I've updated the PR accordingly.